hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/Dispatcher.java

```diff
@@ -164,7 +164,7 @@ synchronized int allocate(int n) {
       }
     }
 
-    /** Aloocate a single lot of items */
+    /** Allocate a single lot of items */
```

Comment:
Could you please also update the method name `Dispatcher#getBblocksMoved` to `Dispatcher#getBlocksMoved`? Thank you.